src/format/FormatFactory.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Format;
+
+class FormatFactory
+{
+  public $data;
+
+  function __construct($data)
+  {
+    $this->data = $data;
+  }
+
+
+  public function create()
+  {
+    $getFormat = 'App\Format\Format' . ucfirst($_GET['format']);
+    $handleFormat = new $getFormat($this->data);
+    $this->data = $handleFormat->formatData();
```

Comment:
try always to use the concept of "return early", Don't set a variable and then return that variable and  I am not going to use this variable anywhere else.